### ISSUE TITLE
Curious what the timezone does?

### ISSUE THREAD

**Issue by professor-farnsworth** [linuxserver/Clarkson#22]
The readme here and for the docker container both say to specify the timezone, but I'm curious what that controls?  I'm wondering because I set it to "America/Los_Angeles" and when I add a fill, the time gets recorded as the current UTC time.  I thought this might be a bug or something I messed up in setting things up....  Or is this expected behavior that everything is recorded in UTC?

Thanks for the help in advance!

**Comment by JoshStark** [comment]
I don't think the TZ really has any effect on the app itself, but it sets up the container's internal timezone which can sometimes affect running applications which query it.

**Comment by professor-farnsworth** [comment]
Yeah, I figured it didn't do much.  All the docker images from linuxserver.io include a TZ env variable, so I guess they just toss that out there for everything.

So it's expected behavior then that everything is recorded in UTC?  That's fine with me, just wanted to make sure I didn't miss something.
